spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET /users' do
+    it 'renders index template with correct placeholder text for users' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
+      expect(response.body).to include('Here are lists of users')
+    end
+  end
+
+  describe 'GET /users/:id' do
+    it 'renders show template with correct placeholder text' do
```

Comment:
* Kindly add also a test that checks If the response body includes the correct placeholder text as its a requirement 
![image](https://user-images.githubusercontent.com/52497246/223382196-3fe53fcc-66c6-443d-8242-c59963ef4156.png)